torch/_meta_registrations.py

```diff
@@ -3797,6 +3797,12 @@ def meta_masked_scatter_backward(self, mask, sizes):
     return self.new_empty(sizes)
 
 
+@register_meta(aten._masked_scale.default)
+def meta__masked_scale(self, mask, scale):
+    masked_scale = self.new_empty(self.size())
```

Comment:
This should be created with `memory_format=utils.suggest_memory_format(self)`.

You also need to rebase to fix some of the flaky tests, but many are genuine test failures caused by tests marked as xfail passing.  You need to update the xfails.

Reply:
You still need to fix the xpasses.